frontend/src/components/App.js

```diff
@@ -57,6 +121,10 @@ class App extends React.Component {
         this.setState({start: date});
     };
 
+    handleCalendar = (calendar) => {
+        this.setState({calendar: calendar});
```

Comment:
We can even make it shorter using object property value shorthand this.setState({ calendar })